interface/main/calendar/add_edit_event.php

```diff
@@ -2028,6 +2028,8 @@ function are_days_checked(){
 * this enable to add new rules for this form in the pageValidation list.
 * */
 var collectvalidation = <?php echo($collectthis); ?>;
+var g_date_format='<?php echo $GLOBALS["date_display_format"];?>';
```

Comment:
hi @eyalvo-matrix ,
Isn't this already being set in validation_script.js.php that is included in this script?

Reply:
As I can see add_edit_event.php only loads  validate_extend.js and validate_modified.js

Reply:
I'm seeing it at line 106

Reply:
Yes I see that, and It is hard to track the code but as far that I can see it is not defined in the popup
I ran the following test 
![image](https://user-images.githubusercontent.com/20621444/37449901-aaa6b774-2834-11e8-8766-d3daf7be2ed9.png)

Reply:
Looks like there is a bug in validation_script.js.php that needs to be addressed. If I look at the source from browser I see:
``````
var g_date_format='0';
``````

Reply:
oh wait, actually, that is correct, it should be set to 0

Reply:
So, the value is getting set in the current codebase.

Reply:
Ummm ...Strange I will do a new installation and re-check it 
maybe somehow I did not get the latest pull

Reply:
I think it is an ordering issue in validation_script.js.php
Note that validate_extend.js is being included before the g_date_format variable is set